test/cli/samples/basic/expected/Main.js

```diff
@@ -1,4 +1,4 @@
-/* src/Main.html generated by Svelte v2.5.1 */
+/* src/Main.html generated by Svelte v2.6.6 */
```

Comment:
I thought we updated the CLI tests before they were merged in to replace the version number with x.y.z - when did that go away?

Reply:
i think we just changed the tests so that it would replace the version numbers when it compared the actual vs expected. would probably be better if it did that when the file is written, to avoid cluttering diffs